cadc-data-ops-fits/src/main/java/org/opencadc/fits/FitsOperations.java

```diff
@@ -130,4 +131,22 @@ public Header getPrimaryHeader() throws ReadException {
             throw new ReadException("failed to read " + src.getAbsolutePath(), ex);
         }
     }
+
```

Comment:
Not sure about the method name or the `String cutoutSpec` arg, but also don't have anything better to suggest right now. I know that just passing String around ends up being a total non-type-safe mess eventually... We'll eventually want to separate parsing/validating string input from this method, but that will happen later on.